hack/make-project-vars.mk

```diff
@@ -5,7 +5,7 @@ ENVTEST_ASSETS_DIR := $(PROJECT_DIR)/testbin
 GOBIN ?= $(BIN_DIR)
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
-GOPROXY ?= direct,https://proxy.golang.org
+GOPROXY ?= https://proxy.golang.org/
```

Comment:
atleast add `direct` to the end, this might fail if the pkg isn't found the proxy afair

Reply:
No, it never failed; it was the same earlier. First of all, if you add something to this module, you will make the changes on your local machine, and pulling the new module on your local machine will make it available on the proxy server, provided you are the first one to pull it. Otherwise it will be already present on the proxy server.